### DISCUSSION TITLE
Is there a way to log errors when using extend with JS option

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3573] [Q&A] [answered]
i am writing a cronjob inside that I am fetching records, but somehow it can't fetch the data, i want to know why it is unable to fetch (some kind of logs)
2nd question: is there a way to create customModel

**Comment by ganigeorgiev** [comment, marked as answer]
Out of the box at the moment you can only use the standard `console.log/error` to log to the stdout/stderr output.

Custom application level logs are planned for v0.20/v0.21 as part of the "Logs generalization" task.

If you want to store the logs in the database, as a workaround you can also create "logs" collection with a `json` field, and on `try/catch` error inside your cron task you can create a new record in that collection.

> is there a way to create customModel

I don't understand what do you mean by this.
